### PR TITLE
Improve resource-path-generators.md

### PR DESCRIPTION
**Pull request by dunglas**

### CODE REVIEW

File: core/resource-path-generators.md [hunk 1]
```diff
@@ -1,24 +1,26 @@
 # Resource path generators
```
Comment (teohhanhui) [inline core/resource-path-generators.md]: From the perspective of a reader I don't think the chapter title is very relatable...

File: core/resource-path-generators.md [hunk 2]
```diff
@@ -1,24 +1,26 @@
 # Resource path generators
 
-With API Platform, you can configure the path of your exposed resources. 
+With API Platform, you can configure globally the path of your exposed resources.
```
Comment (teohhanhui) [inline core/resource-path-generators.md]: When I read this sentence, what comes to mind is the root prefix (e.g. `/api`) :laughing:

File: core/resource-path-generators.md (hunk 1 above)
Comment (dunglas) [inline core/resource-path-generators.md]: WDYT of my last commit?

Comment (teohhanhui) [inline core/resource-path-generators.md]: Better :+1:

Comment (dunglas) [inline core/resource-path-generators.md]: Maybe should we also rename those services in the code?

Comment (teohhanhui) [inline core/resource-path-generators.md]: It was @sroze's idea: https://github.com/api-platform/core/pull/547#discussion_r64135705

File: core/path-naming-strategy.md [hunk 3]
```diff
@@ -1,38 +1,37 @@
-# Resource path generators
+# Path naming strategy
 
-With API Platform, you can configure the path of your exposed resources. 
+With API Platform, you can configure globally the naming strategy of URL paths exposing your resources.
```
Comment (soyuka) [inline core/path-naming-strategy.md]: `globally` should be at the end of the sentence

**Pull request merged**